src/features/auth/pages/signup/SignupForm.web.test.tsx

```diff
@@ -17,39 +14,45 @@ jest.mock('uuid', () => {
   }
 })
 
-const defaultProps = {
-  navigation,
-  route: { name: 'SignupForm', key: '', params: { preventCancellation: false } },
-} as unknown as StackScreenProps<RootStackParamList, 'SignupForm'>
-
 const realUseState = React.useState
 const mockUseState = jest.spyOn(React, 'useState')
 
 jest.spyOn(useFeatureFlagAPI, 'useFeatureFlag').mockReturnValue(false)
 
 describe('<SignupForm/>', () => {
   describe('Accessibility', () => {
+    it('should not have basic accessibility issues for SetEmail', async () => {
+      const { container } = render(
+        <SafeAreaProvider>
+          <SignupForm />
+        </SafeAreaProvider>
+      )
+      await waitFor(() => {
+        expect(screen.getByTestId('Entrée pour l’email')).toHaveFocus()
```

Comment:
On avait noté ce "hack" quelque part dans une doc ?

Reply:
Je me suis mis une action perso pour faire une doc avec tous les troubleshooting de test sur lesquels on a pu tomber, je vais essayer de présenter ça en synchro test la semaine pro ! (Mais bonne remarque en tout cas)